mongoScripts/mappingData.js

```diff
@@ -0,0 +1,17 @@
+var db = connect('localhost:27017/db')
+
+var users = db.users.find();
+
+var address1 = db.addresses.find();
+
+var address2 = db.addresses.find().skip(50);
+
+
+
+while (users.hasNext()) {
```

Comment:
What do we want to do here?

Reply:
im trying to distribute each user 2 addresses

Reply:
just to create database thats all , nothing to do with main motive of assignment